tdrs-backend/tdpservice/stts/models.py

```diff
@@ -37,6 +37,7 @@ class EntityType(models.TextChoices):
     stt_code = models.PositiveIntegerField(blank=True, null=True)
```

Comment:
`stt_code` should be a string since we need to retain leading zeros as shown [here](https://github.com/raft-tech/TANF-app/tree/develop/tdrs-backend/tdpservice/stts/management/commands/data), but this can be handled in a later ticket since its not pertinent here, maybe #217

Reply:
Yes, this field was added a while ago and isn't scope for this ticket -- I'll make a separate ticket.